_includes/docs/v2.23/resource-coverage.md

```diff
@@ -44,7 +44,7 @@
 | [Compute Engine Instance Templates](https://cloud.google.com/compute/docs/reference/latest/instanceTemplates) |
 | [Compute Engine Interconnects](https://cloud.google.com/compute/docs/reference/rest/v1/interconnects) |
 | [Compute Engine Interconnect Attachments](https://cloud.google.com/compute/docs/reference/rest/v1/interconnectAttachments) |
-| [Compute Engine Licenses](https://cloud.google.com/compute/docs/reference/rest/v1/licenses/get) |
+| Compute Engine Licenses |
```

Comment:
I do see this 404 publicly.  Is this a temporary problem on their end, but will come back?  I still see it in the search result.

Reply:
According to the Compute Engine team, the API is not changing, but they are removing the public documentation because it is not useful for end users and has caused confusion.

Reply:
Sounds good.